docs/decisions/0004-events-status.rst

```diff
@@ -0,0 +1,60 @@
+1. Events status
+================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Each Open edX Event will evolve according to the needs of the community.
+For that reason, with this ADR, we will define a lifecycle that each
+event will follow individually.
+
+Decision
+--------
+
+Each Open edX Event will follow the following lifecycle:
+
+State 1. Provisional
+~~~~~~~~~~~~~~~~~~~~
+
+Events just created and accepted in the repository `openedx-events`,
+these events haven't been accepted in Open edX platform or by the community.
+
+State 2. Active
+~~~~~~~~~~~~~~~
+
+Events being used by Open edX platform and by the community.
+
+State 3. Deprecated
+~~~~~~~~~~~~~~~~~~~
+
+Events that members of the community decide to deprecate in Open edX platform.
+
+State 4. Removed
+~~~~~~~~~~~~~~~~~
+
+Events that members of the community removed from Open edX platform after
+documentation and discussion of the removal.
```

Comment:
Once we get to removal, wouldn’t it just be deleted along with its documentation? Would documentation for this (as well as other state changes) be captured in the changelog?